common/kafka/tests/client_config.properties

```diff
@@ -0,0 +1,10 @@
+# Comment Java kafka client config file.
+# Following is empty line without any white space.
+
+enable.sparse.connections=true
+
+timeout_millis=1200
+socket_timeout_ms=300
+
```

Comment:
Can you add a commented line to the test?

Reply:
👍